### ISSUE TITLE
TypeError: Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by Farmer-cb** [iobroker-community-adapters/ioBroker.proxmox#50]
sporadisch kommt folgende Fehlermeldung

`

proxmox.0 | 2020-02-28 17:18:22.759 | error | at IncomingMessage.emit (events.js:203:15)
-- | -- | -- | --
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Object.onceWrapper (events.js:286:20)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1083:12)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Request.emit (events.js:198:13)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Request.<anonymous> (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1161:10)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Request.emit (events.js:198:13)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Request.self.callback (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:185:22)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at Request.request [as _callback] (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
proxmox.0 | 2020-02-28 17:18:22.759 | error | at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33
proxmox.0 | 2020-02-28 17:18:22.759 | error | (422) TypeError: Cannot read property 'name' of undefined
proxmox.0 | 2020-02-28 17:18:22.759 | error | (422) uncaught exception: Cannot read property 'name' of undefined

`

**Comment by xXBJXx** [comment]
ich habe den Error auch.
Ich habe bei mir einen Nas in Proxmox eingebunden für die Backups wen ich den Nas neustarte oder er nicht mehr erreichbar ist bekomme ich den Error und der Adapter stürzt solange ab bis das Nas wieder da ist.

`

host.Master | 2020-03-06 21:53:37.280 | info | Restart adapter system.adapter.proxmox.0 because enabled
-- | -- | -- | --
host.Master | 2020-03-06 21:53:37.280 | info | instance system.adapter.proxmox.0 terminated with code 0 (NO_ERROR)
host.Master | 2020-03-06 21:53:37.280 | error | Caught by controller[2]:     at IncomingMessage.emit (events.js:203:15)
host.Master | 2020-03-06 21:53:37.280 | error | Caught by controller[2]:     at Object.onceWrapper (events.js:286:20)
host.Master | 2020-03-06 21:53:37.280 | error | Caught  by controller[2]:     at IncomingMessage.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1083:12)
host.Master | 2020-03-06 21:53:37.280 | error | Caught by controller[2]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.280 | error | Caught  by controller[2]:     at Request.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1161:10)
host.Master | 2020-03-06 21:53:37.280 | error | Caught by controller[2]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.280 | error | Caught  by controller[2]:     at Request.self.callback  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:185:22)
host.Master | 2020-03-06 21:53:37.280 | error | Caught  by controller[2]:     at Request.request [as _callback]  (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[2]:     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[2]:     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[2]: TypeError: Cannot read property 'name' of undefined
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at IncomingMessage.emit (events.js:203:15)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at Object.onceWrapper (events.js:286:20)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[1]:     at IncomingMessage.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1083:12)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[1]:     at Request.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1161:10)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[1]:     at Request.self.callback  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:185:22)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[1]:     at Request.request [as _callback]  (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]:     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[1]: TypeError: Cannot read property 'name' of undefined
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[0]:     at IncomingMessage.emit (events.js:203:15)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[0]:     at Object.onceWrapper (events.js:286:20)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[0]:     at IncomingMessage.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1083:12)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[0]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.279 | error | Caught  by controller[0]:     at Request.<anonymous>  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:1161:10)
host.Master | 2020-03-06 21:53:37.279 | error | Caught by controller[0]:     at Request.emit (events.js:198:13)
host.Master | 2020-03-06 21:53:37.278 | error | Caught  by controller[0]:     at Request.self.callback  (/opt/iobroker/node_modules/iobroker.proxmox/node_modules/request/request.js:185:22)
host.Master | 2020-03-06 21:53:37.278 | error | Caught  by controller[0]:     at Request.request [as _callback]  (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
host.Master | 2020-03-06 21:53:37.278 | error | Caught by controller[0]:     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
host.Master | 2020-03-06 21:53:37.278 | error | Caught by controller[0]:     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33
host.Master | 2020-03-06 21:53:37.278 | error | Caught by controller[0]: TypeError: Cannot read property 'name' of undefined
proxmox.0 | 2020-03-06 21:53:36.725 | info | (14708) Terminated (NO_ERROR): Without reason
proxmox.0 | 2020-03-06 21:53:36.725 | info | (14708) terminating
proxmox.0 | 2020-03-06 21:53:36.722 | error | (14708) TypeError: Cannot read property 'name' of undefined     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/pro
proxmox.0 | 2020-03-06 21:53:36.722 | error | (14708) uncaught exception: Cannot read property 'name' of undefined
proxmox.0 | 2020-03-06 21:53:36.722 | error | (14708) TypeError: Cannot read property 'name' of undefined     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/pro
proxmox.0 | 2020-03-06 21:53:36.722 | error | (14708) uncaught exception: Cannot read property 'name' of undefined
proxmox.0 | 2020-03-06 21:53:36.721 | info | (14708) cleaned everything up...
proxmox.0 | 2020-03-06 21:53:36.721 | error | (14708) TypeError: Cannot read property 'name' of undefined     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:415:33     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/pro
proxmox.0 | 2020-03-06 21:53:36.721 | error | (14708) uncaught exception: Cannot read property 'name' of undefined

`

**Comment by Feuer-sturm** [comment]
Ich habe die proxmox Adapter Version 1.0.1 im Einsatz und auch mein NAS in proxmox für Backups eingebunden. Habe die gleichen Fehlermeldungen wie ihr auch, wenn ich mein NAS ausschalte.
Wenn das NAS wieder online ist, kommen auch keine Meldungen mehr.

```
proxmox.0 | 2020-03-07 11:02:48.878 | info | (19886) starting. Version 1.0.1 in /opt/iobroker/node_modules/iobroker.proxmox, node: v10.18.1
-- | -- | -- | --
host.beebox(iobrokerMaster) | 2020-03-07 11:02:46.928 | info | instance system.adapter.proxmox.0 started with pid 19886
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.895 | info | Restart adapter system.adapter.proxmox.0 because enabled
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.895 | info | instance system.adapter.proxmox.0 terminated with code 0 (NO_ERROR)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.895 | error | Caught by controller[0]:     at IncomingMessage.emit (events.js:203:15)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at Object.onceWrapper (events.js:286:20)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/request/request.js:1083:12)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at Request.emit (events.js:198:13)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at Request.<anonymous> (/opt/iobroker/node_modules/request/request.js:1161:10)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at Request.emit (events.js:198:13)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.894 | error | Caught by controller[0]:     at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.893 | error | Caught by controller[0]:     at Request.request [as _callback]  (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.893 | error | Caught by controller[0]:     at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.893 | error | Caught by controller[0]:     at /opt/iobroker/node_modules/iobroker.proxmox/main.js:467:74
host.beebox(iobrokerMaster) | 2020-03-07 11:02:16.892 | error | Caught by controller[0]: TypeError: Cannot read property 'name' of undefined
proxmox.0 | 2020-03-07 11:02:16.356 | info | (19871) Terminated (NO_ERROR): Without reason
proxmox.0 | 2020-03-07 11:02:16.354 | info | (19871) terminating
proxmox.0 | 2020-03-07 11:02:16.346 | info | (19871) cleaned everything up...
proxmox.0 | 2020-03-07 11:02:16.344 | error | at IncomingMessage.emit (events.js:203:15)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Object.onceWrapper (events.js:286:20)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at IncomingMessage.<anonymous> (/opt/iobroker/node_modules/request/request.js:1083:12)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Request.emit (events.js:198:13)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Request.<anonymous> (/opt/iobroker/node_modules/request/request.js:1161:10)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Request.emit (events.js:198:13)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Request.self.callback (/opt/iobroker/node_modules/request/request.js:185:22)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at Request.request [as _callback] (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:166:21)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at _get.then.data (/opt/iobroker/node_modules/iobroker.proxmox/lib/proxmox.js:48:9)
proxmox.0 | 2020-03-07 11:02:16.344 | error | at /opt/iobroker/node_modules/iobroker.proxmox/main.js:467:74
proxmox.0 | 2020-03-07 11:02:16.344 | error | (19871) TypeError: Cannot read property 'name' of undefined
proxmox.0 | 2020-03-07 11:02:16.343 | error | (19871) uncaught exception: Cannot read property 'name' of undefined
```

**Comment by MeisterTR** [comment]
should be fixed in 1.0.2

**Comment by Feuer-sturm** [comment]
Ich hab jetzt Version 1.0.2 installiert und mein NAS ausgeschaltet. Es kommen keine Fehlermeldungen mehr :-)
